src/modules/Channel/context/hooks/useDeleteMessageCallback.js

```diff
@@ -18,7 +18,7 @@ function useDeleteMessageCallback({ currentGroupChannel, messagesDispatcher }, {
         resolve(message);
       }
```

Comment:
don't we need `else` here?

Reply:
wow I did not know, thank you!
https://stackoverflow.com/questions/28896280/why-does-javascript-es6-promises-continue-execution-after-a-resolve